cpp/ql/src/semmle/code/cpp/Variable.qll

```diff
@@ -4,7 +4,20 @@ import semmle.code.cpp.Initializer
 private import semmle.code.cpp.internal.ResolveClass
 
 /**
- * A C/C++ variable.
+ * A C/C++ variable. For example in the following code there are four
+ * variables, `a`, `b`, `c` and `d`:
+ * ```
+ * extern int a;
+ * int a;
```

Comment:
I'd remove the `extern int a`, since that belongs in a `VariableDeclarationEntry`.

Reply:
Done.